dev-tools/mage/pkg.go

```diff
@@ -144,6 +144,9 @@ func TestPackages(options ...TestPackagesOption) error {
 		args = append(args, "--modules.d")
 	}
 
+	if BeatUser == "root" {
+		args = append(args, "-root-owner")
```

Comment:
Does 1 or 2 `-` matter here?

Reply:
This is passed to `go test` so I think either works.